CASMINST-4411 - add note for ceph crash (#1266)
Cray-HPE/docs-csm: upgrade/1.2/Stage_1.md

    > NOTE: You may need to reset the root password for each node after it is rebooted

    **Known Issues:**
    * It is possible to encounter an error like shown below.  If this is the case, just re-run the same command for the node upgrade and it will pick up at that point and conitune.

      ```text
      ====> REDEPLOY_CEPH ...
      /usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/root/ceph.pub"Number of key(s) added: 1Now try logging into the machine, with:   "ssh 'root@ncn-s003'"
      and check to make sure that only the key(s) you wanted were added.Error EINVAL: Traceback (most recent call last):
      ```

>     * During the storage node rebuild, it is possible that ceph health may report ***HEALTH_WARN 1 daemons have recently crashed***.  This can occur occasionally as part of the shutdown process of the node being rebuilt.  Please refer to the [Dump Ceph Crash Data](operations/../../../operations/utility_storage/Dump_Ceph_Crash_Data.md) located in the Operations/Utility Storage section of the guide.

Reviewer: ```suggestion
    * During the storage node rebuild, it is possible that Ceph health may report ***HEALTH_WARN 1 daemons have recently crashed***.  This can occur occasionally as part of the shutdown process of the node being rebuilt. Please refer to the [Dump Ceph Crash Data](operations/../../../operations/utility_storage/Dump_Ceph_Crash_Data.md) procedure for more information.
```

**IMPORTANT:** Ensure the Ceph cluster is healthy prior to continuing. If you have processes not running, then refer to [Utility Storage Operations](../../operations/utility_storage/Utility_Storage.md) for operational and troubleshooting procedures.

1. Repeat the previous step for each other storage node, one at a time.

> 2. After `ncn-upgrade-ceph-nodes.sh` has successfully run for all storage nodes, rescan SSH keys on all storage nodes

Reviewer: Add a period to the end of the line

Reviewer 2: They merged before fixing this. I'll address it in [CASMINST-4423](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4423)
